migrations/versions/0d412b939e30_add_additional_state_columns.py

```diff
@@ -0,0 +1,46 @@
+"""Add additional state columns
+
+Revision ID: 0d412b939e30
+Revises: 51fcfb4b5daa
+Create Date: 2020-05-18 23:03:54.641232
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0d412b939e30'
+down_revision = '51fcfb4b5daa'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
one thing I missed in my previous PR is a migration that renames e.g. dataDate - do you mind regenerating this migration to pick up both?

Reply:
I created a new migration for it since I already ran the previous one locally and Alembic gets shouty about changing existing migrations unless you trick it.